ReadMe.md

```diff
@@ -77,7 +86,12 @@ npm install
 
 This will install all the required `node_modules` for the VueJS frontend, including any necessary libraries for building the interface and making API requests to the backend.
 
- Step 4: Run the Frontend
+ Step 4: Create a .env file in the frontend folder and add the following content in it:
```

Comment:
Same. We want to actually create these files & logic now for dev & production. 

Production will require more thought as you'll need to configure & modify the code so that it can execute on a real server & interact with real endpoints.

Reply:
When we move towards production on an actual endpoint, we would need to change the VUE_APP_API_URL in this .env file to point to that endpoint URL.